recipe/meta.yaml

```
@@ -26,35 +25,40 @@ build:
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
+    # We need CXX during build time to generate rbconfig.rb
     - {{ compiler('cxx') }}
-    - autoconf    # [unix]
-    - automake    # [unix]
-    - make        # [unix]
-    - libtool     # [unix]
+    - gnuconfig                        # [unix]
+    - autoconf                         # [unix]
+    - automake                         # [unix]
+    - libtool                          # [unix]
+    - make                             # [unix]
     - pkg-config
-    - patch         # [not win]
-    - m2-patch      # [win]
+    - bison                            # [unix]
+    - gperf                            # [unix]
   host:
-    - gdbm 1.18     # [x86 and unix]
-    - gettext 0.21  # [osx]
-    - gmp 6.2.1     # [unix]
-    - libffi 3.4
-    - ncurses 6.4   # [unix]
     - openssl {{ openssl }}
-    - readline 8.2  # [unix]
+    # libyaml (for psych)
     - yaml 0.2.5
     - zlib {{ zlib }}
+    - libffi 3.4
+    - readline 8.2                     # [unix]
+    # need for readline
+    - ncurses 6.4                      # [unix]
+    - gettext 0.21                     # [osx]
+    - gmp 6.2.1                        # [unix]
+    # ext/gdbm (old db need for old tests and old projects)
+    - gdbm 1.18                        # [unix]
```

```suggestion
    - libffi {{ libffi }}
    - readline {{ readline }}          # [unix]
    # need for readline
    - ncurses {{ ncurses }}            # [unix]
    - gettext {{ gettext }}            # [osx]
    - gmp 6.2.1                        # [unix]
    # ext/gdbm (old db need for old tests and old projects)
    - gdbm {{ gdbm }}                  # [unix]
```

`gmp` is pinned in the cbc at 6.3.0. Do we need this older version here?

---

Those pins was inherited, so it might have sense, but no any comments about that.
Let's try get them from cbc matrix. Thank you